include/func_tables.h

```diff
@@ -24,6 +24,7 @@ void func_80016E34(void);
 void func_80016E84(void);
 
 // D_800F1F64
+extern void (*D_800F1F64[])(struct Unk* arg0);
```

Comment:
Could be that `Unk` and `EngineObj` are the same?

Reply:
Unk is a a bunch of objects mixed together since I couldn't tell before adding the object trace stuff. In this case since func_800193D8 uses this and passes in EngineObj it looks like that whole table should be EngineObj. I made a scratch for the first function and it is using arg0 https://decomp.me/scratch/oSJra